### PR TITLE
Change all instances and examples of Device to System

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates the documentation in line with https://github.com/dronecore/DroneCore/pull/310

### CODE REVIEW

**Comment by hamishwillee** [comment]
@shakthi-prashanth-m This updates the docs in a fairly straightforward manner, so should mostly be correct. What I noticed though is that we really need examples of how to use the new has_* methods. When do you think the camera stuff will be ready?

File: en/guide/system_information.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+std::cout << " UUID: " << info->uuid() << std::endl;
+
+// Wait until version/firmware information has been populated from the vehicle
+while (!info->is_complete()) {
+    std::cout << "Waiting for Version information to populate from system." << std::endl;
+    std::this_thread::sleep_for(std::chrono::seconds(1));
+}
+
+// Get the system Version struct
+const Info::Version &systemVersion =  info->get_version();
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/guide/system_information.md]: I just replaced the previous pattern :-). Don't see this as urgent!

File: en/SUMMARY.md [hunk 2]
```diff
@@ -5,9 +5,9 @@
 * [Quick Start](getting_started/README.md)
 * [Guide](guide/README.md)
   * [Usage/Paradigms](guide/general_usage.md)
-  * [Connecting to Devices](guide/connections.md)
-  * [Managing Devices](guide/using_plugins.md)
-  * [Vehicle Information](guide/device_version.md)
+  * [Connecting to Systems (Vehicles)](guide/connections.md)
+  * [Managing Systems](guide/using_plugins.md)
+  * [System Information](guide/system_information.md)
```
[the remark this answers is not in the document]
Comment (hamishwillee) [reply en/SUMMARY.md]: These links all work when built.

**Comment by hamishwillee** [comment]
@shakthi-prashanth-m Thanks, I've merged this.

**Comment by shakthi-prashanth-m** [comment]
@hamishwillee 
>What I noticed though is that we really need examples of how to use the new has_* methods.

Sure I agree, will add them after I am done with camera work.
 
>When do you think the camera stuff will be ready?

I've locally re-based camera with latest changes in core and adding video streaming interface on top of it. It should take appx 2 days. @julianoes and me planned to merge basic functionality of camera first.
And, @julianoes will add new PR for adding changes related to camera settings.

**Comment by hamishwillee** [comment]
Thanks @shakthi-prashanth-m ! FYI, support for video capture just went into CSD.

**Comment by shakthi-prashanth-m** [comment]
Yeah, Thanks! I am in sync with Lalit :-)

**Pull request merged**